### PR TITLE
Fix for .travis.yml to not shadow usable creds, allowing compile to succeed (C4-201)

### PR DESCRIPTION
**Pull request by netsettler**
We already debugged this in another couple of PRs but I wanted to isolate this so it was easy to approve and merge.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ env:
...
+#  - secure: z6WQVmKFTWf7in9J8764ic6gs5QyLctGYIDzstjSBXYrLTUfJjUwB7OAup1fAFXYUxtzkn6vjn3CIuy+5Nq7jL25Bwen4SJCPZtk9w/8acXUNDNlIFIak7T/jelOO11gHxHI6j2H+wj1BGxz1XZEkx8eeUN6389yvWgHQH01sbrQ7gXRgfmK+1pvUK9Li+4Mn9gBbOoqDxz2w8Wuazkk/FcmlYRYet4z8/y4FHwyc+Sa69utVJQAhDwJzngtb5WRdlWlrcFfyWhQ0eZ/shKxBcjekU5ZeLCh0av4ttB2IXVvLHzsr6vuCssRJqC7LHqiWEpXacWSf0ZOpMlAzwT+fPH5xXSl1kdc8vkpz8gYC5YDDMxkTrUoRv8vCYi/KWmYXNzdiAQn53aX741fcj5FV8v0ybmdPPQLfeQPDhCeHLlroWZsI5cgygxotIfeQzOY3MXSi+YqtZjmuHjAC1aINn9sLwYtCbV1ZVn3TxPaMyWjPlBuOaYoU6GkC+HxAcUy9QMVxBBzbVbOnaPri8NTXxHE58cIz+6GZVou4UDpEjjpi3Z8hieXu5cR0KKFVPofIQlgLaFZ9YgtAuXW6RP2k7ONYyCbW7bz39nnY6outFe96yR/HB4UKqBwc8dIUPfgzyoEyE1obnuNcsNf6UZf+B5ALM03ejQCx9KJuFPrZSw=
```
Comment (willronchetti) [inline .travis.yml:8]: Can we remove these?

Comment (netsettler) [reply .travis.yml:8]: I didn't try. I was going to test that separately. This was the minimal change to unshadow the two variables we made in the travis repo settings. I don't know what the values are to put in the settings if it fails with these removed.

Comment (netsettler) [reply .travis.yml]: OK, I did the experiment and it was fine, so I merged that change. I'll merge this.

**Comment by netsettler** [comment]
Sarah's auth PR went in ahead of this. It had a slight variation of this fix incorporated so has technically fixed what this set out to fix, though there are some extra details (like removing of the secrets) that weren't in that one, so I'll go ahead and merge this as well.

**Pull request merged**